docs/blog/2022-02-02-Changes_to_Slurm_srun_for_interactive_jobs.md

```diff
@@ -0,0 +1,15 @@
+# Changes to Slurm "srun" for Interactive Jobs
+
+During the recent system time the Slurm job scheduler was upgraded. 
```

Comment:
Could put the date of the system time instead of "recent"

Reply:
I don't have the exact date of the system time. Can you put it here? Additionally, doesn't the date on the Blog more or less takes care of it?

Reply:
Most recent commit should fix this.